docs-ref-conceptual/azure-cli-reference-for-IoT.md

```diff
@@ -45,14 +44,13 @@ The IoT **extension** introduces rich features and functionality to manage, mani
 | [az iot dps](/cli/azure/ext/azure-iot/iot/dps) | yes | Manage entities in an Azure IoT Hub Device Provisioning Service.
 | [az iot edge](/cli/azure/ext/azure-iot/iot/edge) | | Manage IoT solutions on the Edge.
 | [az iot hub](/cli/azure/ext/azure-iot/iot/hub) | yes | Manage entities in an Azure IoT Hub.
-| [az iot pnp](/cli/azure/ext/azure-iot/iot/pnp) | | Manage entities of an IoT Plug and Play model repository.
 
 ### Additional CLI commands for Azure services used by IoT
 
 | Reference | Type | Description
 |-|-|-|
 | [az maps](/cli/azure/maps) | core | Manage Azure Maps.
-| [az timeseriesinsights](/cli/azure/ext/timeseriesinsights/timeseriesinsights) | extension | Manage Azure Time Series Insights.
+| [az timeseriesinsights](/cli/azure/ext/timeseriesinsights) | extension | Manage Azure Time Series Insights.
```

Comment:
**az timeseriesinsights** is now **az tsi** although the old name still works in the URL.

![image](https://user-images.githubusercontent.com/58762114/111840950-91c4c180-88d3-11eb-88a1-1d64f3f1b738.png)

Reply:
updated